rbac/server/api/users.py

```diff
@@ -433,6 +680,21 @@
 
 
 @USERS_BP.get("api/users/<next_id>/proposals/open")
+@doc.summary("Get open proposals for a user, by their next_id.")
+@doc.description("Get open proposals for a user, by their next_id.")
+@doc.consumes({"head": str}, location="query")
+@doc.consumes({"start": int}, location="query")
+@doc.consumes({"limit": int}, location="query")
+@doc.produces(
+    [{"id": str, "type": str, "object": str, "target": str, "approvers": [str]}],
```

Comment:
200 response appears to be missing link, paging, the list of proposal objects, and head. Here's one response from the API:
```
{
  "link": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/open/?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
  "paging": {
    "last": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/open/?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
    "next": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/open/?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
    "first": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/open/?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
    "limit": 100,
    "total": 2,
    "start": 0,
    "prev": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/open/?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100"
  },
  "data": [
    {
      "close_reason": "",
      "target": "9ffb601a-3fcf-4614-aeff-5a6b27551025",
      "id": "7f755280-e0e2-42db-a9fa-f1050d4b7799",
      "pack_id": "",
      "assigned_approver": [
        "9ffb601a-3fcf-4614-aeff-5a6b27551025"
      ],
      "approvers": [
        "9ffb601a-3fcf-4614-aeff-5a6b27551025"
      ],
      "created_date": 1565042061,
      "object": "069e0083-5378-4b21-939a-5323c1f8a159",
      "status": "OPEN",
      "open_reason": "",
      "metadata": {},
      "closer": "",
      "opener": "9ffb601a-3fcf-4614-aeff-5a6b27551025",
      "type": "UPDATE_USER_MANAGER"
    }
  ],
  "head": "5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526"
}
```

Reply:
we should probably write techdebt tickets to update our response forming function docstrings and endpoint docstrings at some point. The fact that we need to hit our endpoints to get complete payload schemas, and that they don't match our current docstrings is probably going to become a pain point later if it's not already one

Reply:
Sounds good to me. Should we just create one ticket to track this for now?

Reply:
I've created ticket #631 if you would like to add more to it. Please go ahead and resolve this conversation if all is well.